src/java/org/apache/cassandra/service/accord/AccordVerbHandler.java

```diff
@@ -43,6 +43,19 @@ public AccordVerbHandler(Node node)
     public void doVerb(Message<T> message) throws IOException
     {
         logger.debug("Receiving {} from {}", message.payload, message.from());
-        message.payload.process(node, EndpointMapping.getId(message.from()), message);
+        T request = message.payload;
+        Node.Id from = EndpointMapping.getId(message.from());
```

Comment:
Move this into the innermost `if` since it is unused outside.

Reply:
`from` is needed on line 59 as well, so if the outer if is false, then we do `request.process(node, from, message);`